Mathlib/Algebra/Field/Defs.lean

```diff
@@ -47,7 +47,7 @@ field, division ring, skew field, skew-field, skewfield
 assert_not_exists NeZero
 
 -- Check that we have not imported `Mathlib.Tactic.Common` yet.
-assert_not_exists Mathlib.Tactic.LibrarySearch.librarySearch
+assert_not_exists Mathlib.Tactic.scopedNS
```

Comment:
I wonder whether there should be an `assert_not_imported` command for this sort of checks.

Reply:
So do I - but I also think this can be a follow-up change. (Would you like to create a zulip thread or file an issue to record this? That is not something I will insist on.)

Reply:
I started [this Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there-code-for-X.3F/topic/assert_not_imported) about this issue.